### PR TITLE
airbyte-server: tag span of additional OAuth routes

### PR DESCRIPTION
**Pull request by alafanechere**
## What
I forgot to add tags from additional OAuth routes after this initial [PR](https://github.com/airbytehq/airbyte/pull/20468/files#diff-a51e8edc400f0098044b097b2548f045beef41ad99958dcbe3be3dd6d19e2ca8).

## How
Tag the root span with definition and workspace id on `complete_oauth` + `get_consent_url` for sources and destinations.

### CODE REVIEW

File: airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java [hunk 1]
```diff
@@ -210,8 +214,10 @@ public Map<String, Object> completeSourceOAuth(final CompleteSourceOauthRequest
 
   public Map<String, Object> completeDestinationOAuth(final CompleteDestinationOAuthRequest completeDestinationOAuthRequest)
       throws JsonValidationException, ConfigNotFoundException, IOException {
-    ApmTraceUtils.addTagsToTrace(Map.of(WORKSPACE_ID_KEY, completeDestinationOAuthRequest.getWorkspaceId(), DESTINATION_DEFINITION_ID_KEY,
-        completeDestinationOAuthRequest.getDestinationDefinitionId()));
+    final Map<String, Object> traceTags = Map.of(WORKSPACE_ID_KEY, completeDestinationOAuthRequest.getWorkspaceId(), SOURCE_DEFINITION_ID_KEY,
```
Comment (pedroslopez) [inline airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java:217]:
```suggestion
    final Map<String, Object> traceTags = Map.of(WORKSPACE_ID_KEY, completeDestinationOAuthRequest.getWorkspaceId(), DESTINATION_DEFINITION_ID_KEY,
```

**Review by pedroslopez — CHANGES_REQUESTED** [review]

File: airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java (hunk 1 above)
Comment (alafanechere) [reply airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java:217]: Always the same typo 😮‍💨

File: airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java [hunk 2]
```diff
@@ -162,8 +164,10 @@ public OAuthConsentRead getDestinationOAuthConsent(final DestinationOauthConsent
 
   public Map<String, Object> completeSourceOAuth(final CompleteSourceOauthRequest completeSourceOauthRequest)
       throws JsonValidationException, ConfigNotFoundException, IOException {
-    ApmTraceUtils.addTagsToTrace(Map.of(WORKSPACE_ID_KEY, completeSourceOauthRequest.getWorkspaceId(), SOURCE_DEFINITION_ID_KEY,
-        completeSourceOauthRequest.getSourceDefinitionId()));
+    final Map<String, Object> traceTags = Map.of(WORKSPACE_ID_KEY, completeSourceOauthRequest.getWorkspaceId(), SOURCE_DEFINITION_ID_KEY,
+        completeSourceOauthRequest.getSourceDefinitionId());
```
Comment (pedroslopez) [inline airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java:168]: should we have some sort of helper like getOAuthTraceTags or getDestinationTraceTags to make sure they're consistent?

Comment (alafanechere) [reply airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java:168]: I'll definitely consider more helper if we trace more things in the future 😄

**Review by pedroslopez — APPROVED** [review]

**Pull request merged**